boilerplate/flyte/end2end/end2end.sh

```diff
@@ -31,4 +31,4 @@ fi
 
 make kustomize
 make end2end_execute
-popd
+popd
```

Comment:
can we add back newlines at some point?  i really like newlines.

Reply:
doesn't have to be this PR.

Reply:
Yes i will add in next iteration of boilerplate